### PR TITLE
core: Don't treat known aliases as default fonts before trying to load them as device fonts

### LINKED ISSUE #14675: Font on web doesn't work anymore.
### Describe the bug

After following the guide on wiki for changing font on web it worked on 8-1-2024
image
but on 9-1-2024 without change anything it stopped working.

### Expected behavior

image

### Content Location

you can test it here.

http://www.cocolaniisland.com

### Affected platform

Self-hosted version

### Operating system

Windows 10

### Browser

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by evilpie**
Fixes #14675

### CODE REVIEW

**Review by Lord-McSweeney — APPROVED** [review]

File: core/src/html/layout.rs [hunk 1]
```diff
@@ -523,11 +522,19 @@ impl<'a, 'gc> LayoutContext<'a, 'gc> {
 
         // TODO: handle multiple fonts for a definition, each covering different sets of glyphs
 
-        // Use the default serif font, when no other matching font was found.
+        // Font name was neither one of the three default fonts nor matched any device font.
+        // As a last resort fallback to the default serif font, but as a slight compatibility
```
Comment (torokati44) [inline core/src/html/layout.rs:526]: I had to read this sentence three times before successfully parsing it... 😅
I read the first part as "As a last resort fallback to the default serif font, <do something that would be a fallback for the default serif font?>".
To me, "fallback" is a noun and in this case I would have written "fall back".
But I not English good, so it's probably okay as is.

Comment (Lord-McSweeney) [reply core/src/html/layout.rs:526]: Alternatively, a comma after "resort" would make it clearer, too.

Comment (evilpie) [reply core/src/html/layout.rs]: Fallback is a noun. But I just reworded the whole comment, hopefully it's clearer now?

Comment (torokati44) [reply core/src/html/layout.rs]: Yes, it is clear now. Thanks!

**Pull request merged**